PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Forget RADIUS WiFi Network.md

```diff
@@ -0,0 +1,141 @@
+#### Name
+
+Windows - Forget RADIUS WiFi Network | v1.0 JCCG
+
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+# Enter the SSID of the Radius network
+$RadiusSSID = ''
+
+# Change Overwrite to $true if you would like to re-create existing Scheduled Tasks
+# This should only be used if redistributing this command to devices
+$Overwrite = $false
+
+
+#### DO NOT EDIT BELOW THIS LINE ####
+$removeWifips1 = @"
+function Get-WifiProfile {
+    [cmdletbinding()]
+    param
+    (
+         [System.Array]`$Name = `$NULL
+    )
+    Begin {
+        `$list = ((netsh.exe wlan show profiles) -match '\s{2,}:\s') -replace '.*:\s' , ''
+        `$ProfileList = `$List | Foreach-object { [pscustomobject]@{Name = $_ } }
+    }
+    Process {
+         Foreach (`$WLANProfile in `$Name) {
+            `$ProfileList | Where-Object { `$_.Name -match `$WLANProfile }
+         }
+
+    }
+    End {
+         If (`$Name -eq `$NULL) {
+            `$Profilelist
+         }
+    }
+}
+function Remove-WifiProfile {
+    [cmdletbinding()]
+    param
+    (
+         [System.Array]`$Name = `$NULL
+    )
+    begin {}
+    process {
+         Foreach (`$item in `$Name) {
+            `$Result = (netsh.exe wlan delete profile `$item)
+              If (`$Result -match 'deleted') {
+                   "WifiProfile : `$Item Deleted"
+              } else {
+                   "WifiProfile : `$Item NotFound"
+              }
+         }
+    }
+}
+
+Remove-WifiProfile $RadiusSSID
+"@
+function Distribute-JCScheduledTask {
+    param (
+        [boolean]$Overwrite
+    )
+    # Get Current User for JC Commmand
+    $CurrentUser = [System.Security.Principal.WindowsIdentity]::GetCurrent().Name
+
+    # Scheduled Task XML Configuration
+    $ScheduledTaskXML = @"
+<?xml version="1.0" encoding="UTF-16"?>
+<Task version="1.2" xmlns="http://schemas.microsoft.com/windows/2004/02/mit/task">
+  <Settings>
+    <DisallowStartIfOnBatteries>false</DisallowStartIfOnBatteries>
+    <StopIfGoingOnBatteries>true</StopIfGoingOnBatteries>
+    <MultipleInstancesPolicy>IgnoreNew</MultipleInstancesPolicy>
+    <IdleSettings>
+      <StopOnIdleEnd>true</StopOnIdleEnd>
+      <RestartOnIdle>false</RestartOnIdle>
+    </IdleSettings>
+  </Settings>
+  <Triggers>
+    <EventTrigger>
+      <Subscription>&lt;QueryList&gt;&lt;Query Id="0" Path="Microsoft-Windows-WLAN-AutoConfig/Operational"&gt;&lt;Select Path="Microsoft-Windows-WLAN-AutoConfig/Operational"&gt;*[System[Provider[@Name='Microsoft-Windows-WLAN-AutoConfig'] and Task = 24010 and (EventID=8002)]]&lt;/Select&gt;&lt;/Query&gt;&lt;/QueryList&gt;</Subscription>
+    </EventTrigger>
+  </Triggers>
+  <Actions Context="Author">
+    <Exec>
+      <Command>“C:\Windows\SysWOW64\WindowsPowerShell\v1.0\powershell.exe”</Command>
+      <Arguments>-ExecutionPolicy ByPass -File “C:\scripts\removeWifi.ps1”</Arguments>
```

Comment:
```suggestion
      <Arguments>-ExecutionPolicy ByPass -File "C:\scripts\removeWifi.ps1"</Arguments>
```

Reply:
Changed these, I didn't have an issue with the XML with the original quotes, but I updated just in case